e2e/LongHaul/service/Parameters.cs

```diff
@@ -30,13 +30,6 @@ internal class Parameters
             HelpText = "The instrumentation key string for application insights.")]
         public string InstrumentationKey { get; set; } = Environment.GetEnvironmentVariable("APPLICATION_INSIGHTS_INSTRUMENTATION_KEY");
 
-        [Option(
-            'd',
-            "DeviceId",
-            Required = false,
-            HelpText = "The Id of the device to receive the direct method.")]
-        public string DeviceId { get; set; } = "LongHaulDevice1";
```

Comment:
The service app will continuously query to get online devices, so we don't need this arg any more.